client/packages/common/src/ui/components/inputs/Filters/DateFilter.tsx

```diff
@@ -65,24 +66,26 @@ export const DateFilter: FC<{ filterDefinition: DateFilterDefinition }> = ({
     value,
     width: FILTER_WIDTH,
     onChange: handleChange,
-    textFieldProps: {
-      sx: FilterLabelSx,
-    },
     actions: ['clear', 'accept'] as PickersActionBarAction[],
     displayAs,
     ...getMinMaxDates(urlValue, range, minDate, maxDate),
   };
 
-  return displayAs === 'dateTime' ? (
-    <DateTimePickerInput showTime={true} {...componentProps} />
-  ) : (
-    <DateTimePickerInput {...componentProps} />
+  return (
+    <DateTimePickerInput
+      showTime={displayAs === 'dateTime'}
+      {...componentProps}
+    />
   );
 };
 
-const getDateFromUrl = (query: string, range: RangeOption | undefined) => {
+const getDateFromUrl = (
+  query: string,
+  range: RangeOption | undefined,
+  format: string
+) => {
   const value = typeof query !== 'object' || !range ? query : query[range];
-  return DateUtils.getDateOrNull(value);
+  return DateUtils.getDateOrNull(value, format);
```

Comment:
without passing in the format, partially entered dates (e.g. 10/02/0020, on my way to typing 10/02/2025) would get read as 20/02/2010 and mess with the input